contrib/kubespray/script/k8s_generator.py

```diff
@@ -9,6 +9,61 @@
 logger = get_logger(__name__)
 
 
+def get_docker_cache_config_and_mirrors(layout, cluster_config):
```

Comment:
I think this function can be removed. During installation, using config.yml + services-configuration.yaml.template is enough.

Reply:
Removed